src/default-controls.js

```diff
@@ -39,15 +39,47 @@ export class NextButton extends React.Component {
   constructor() {
     super(...arguments);
     this.handleClick = this.handleClick.bind(this);
+    this.nextButtonDisable = this.nextButtonDisable.bind(this);
   }
   handleClick(event) {
     event.preventDefault();
     this.props.nextSlide();
   }
+
+  nextButtonDisable(
```

Comment:
This function name (ending with a verb) makes it sound like it's a function with side effects.

I don't have a suggestion for a better name... what do you think?

Reply:
how about `getNextButtonIsDisabled`?

Reply:
I think you're on the right track, but it might be a bit too wordy now.  I think your original method name would work if you changed `Disable` to `Disabled`.   That'll remove the action implication from the name.  

`nextButtonDisabled` reads more like a statement/question (which is good for boolean returning functions), and it even reads well in an if statement (e.g. `if (nextButtonDisabled(...)) { /* do stuff */ }`).

Reply:
Got it, thanks!